components/camel-casper/src/test/java/org/apache/camel/component/casper/producer/CasperProducerWith_BLOCK_TRANSFERS_OperationTest.java

```diff
@@ -0,0 +1,70 @@
+package org.apache.camel.component.casper.producer;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Produce;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.casper.CasperConstants;
+import org.apache.camel.component.casper.CasperTestSupport;
+import org.junit.jupiter.api.Test;
+
+import com.syntifi.casper.sdk.model.deploy.executabledeploy.Transfer;
+
+@SuppressWarnings("unchecked")
+class CasperProducerWith_BLOCK_TRANSFERS_OperationTest extends CasperTestSupport {
+	@Produce("direct:start")
+	protected ProducerTemplate template;
+
+	@Override
+	public boolean isUseAdviceWith() {
+		return false;
+	}
+
+	@Test
+	void testCallWithout_parameters() throws Exception {
+		Exchange exchange = createExchangeWithBodyAndHeader(null, CasperConstants.OPERATION, CasperConstants.BLOCK_TRANSFERS);
+		template.send(exchange);
+		Object body = exchange.getIn().getBody();
+		// assert Object is a List
+		assertTrue(body instanceof List);
```

Comment:
I think you can use something like `assertIsInstance` (or a similar one) for this check.

Reply:
fixed